### PR TITLE
fix(APIGuild): change type of `afk_timeout` to allowed values

### PR DESCRIPTION
**Pull request by Mateo-tem**
**Please describe the changes this PR makes and why it should be merged:**
Changes the type of `afk_timeout` to allowed values (60, 300, 900, 1800, 3600)

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**

### CODE REVIEW

**Review by SpaceEEC — APPROVED** [review]

File: rest/v9/guild.ts [hunk 1]
```diff
@@ -238,9 +238,9 @@ export type RESTPatchAPIGuildJSONBody = AddUndefinedToPossiblyUndefinedPropertie
 	 */
 	afk_channel_id?: Snowflake | null;
 	/**
-	 * afk timeout in seconds, can be set to: `60`, `300`, `900`, `1800`, `3600`
```
Comment (vladfrangu) [inline rest/v9/guild.ts:241]: keep the comment

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: rest/v9/guild.ts (hunk 1 above)
Comment (Mateo-tem) [reply rest/v9/guild.ts:241]: Only in `RESTPatchAPIGuildJSONBody` or also in `RESTPostAPIGuildsJSONBody`?

Comment (vladfrangu) [reply rest/v9/guild.ts:241]: Everywhere it was removed

Comment (Mateo-tem) [reply rest/v9/guild.ts:241]: Oh, ok

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**